047-TrafficControlWithDapr/README.md

```diff
@@ -0,0 +1,73 @@
+# What The Hack - TrafficControlWithDapr
+
+## Introduction
+This repository contains several hands-on challenges that will introduce you to [Dapr](https://dapr.io/). You will start with a simple ASP.NET Core application that is composed of several microservices. In each challenge, you'll enhance the application by adding Dapr building blocks and components. At the same time, you'll configure the application to consume Azure-based backing services. When complete, you'll have implemented the following Dapr building blocks:
+
+- Service invocation
+- State-management
+- Publish / Subscribe
+- Bindings
+- Secrets management
+
+As Dapr can run on a variety of hosts, you'll start by running Dapr in self-hosted mode on your computer. Then, you'll deploy the Dapr application to run in Azure Kubernetes Service.
+
+## Learning Objectives
+
+The challenges implement a traffic-control camera system that are commonly found on Dutch highways. Here's how the simulation works:
+
+![Speeding cameras](images/speed-trap-overview.png)
+
+There's 1 entry-camera and 1 exit-camera per lane. When a car passes an entry-camera, a photo of the license plate is taken and the car and the timestamp is registered.
+
+When the car passes an exit-camera, another photo and timestamp are registered. The system then calculates the average speed of the car based on the entry- and exit-timestamp. If a speeding violation is detected, a message is sent to the Central Fine Collection Agency (or CJIB in Dutch). The system retrieves the vehicle information and the vehicle owner is sent a notice for a fine.
+
+## Challenges
+- Challenge 0: **[Install tools and Azure pre-requisites](Student/Challenge-00.md)**
+   - Install the pre-requisites tools and software as well as create the Azure resources required.
+- Challenge 1: **[Run the application](Student/Challenge-01.md)**
+   - Run the Traffic Control application to make sure everything works correctly.
+- Challenge 2: **[Add Dapr service invocation](Student/Challenge-02.md)**
```

Comment:
The challenge titles sound very procedural.  Consider re-naming them to just be the subject they handle.  For example: "Add Dapr service invocation" -> "Dapr Service Invocation" or "Working with Dapr Service Invocation" or something that indicates what a student is going to accomplish in the challenge, like "Implement Synchronous Service calls with Dapr Service Invocation"   

This makes it clearer if someone is trying to get a quick glance of what this hack is about and doesn't know all of the technical terms/features of Dapr.

If you do this, you'll need to update each of the corresponding challenge & solution MD files.

This comment applies for challenges 2-8.